From 4a20f5687d6421c037c427e39269c6793a6c333c Mon Sep 17 00:00:00 2001
From: Jose Lamego <jose.a.lamego@intel.com>
Date: Thu, 23 Aug 2018 11:47:30 -0500
Subject: [PATCH 5/5] add default signature verification policy file

This commit creates a default signature verification policy file
file to allow image and signature validations.

Fixes clearlinux/distribution#160

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---
 policy.json | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 policy.json

diff --git a/policy.json b/policy.json
new file mode 100644
index 00000000..fc643c80
--- /dev/null
+++ b/policy.json
@@ -0,0 +1,11 @@
+{
+    "default": [
+        {
+            "type": "insecureAcceptAnything"
+        }
+    ],
+    "transports":
+        {
+            "docker": {}
+        }
+}
-- 
2.21.0

